frontend/test/ui/test_install.py

```diff
@@ -45,53 +45,53 @@ def test_bottom_install_button(base_url, selenium, firefox, notifications):
         notifications.AddOnInstallComplete).close()
 
 
-# @pytest.mark.skipif(os.environ.get('SKIP_INSTALL_TEST') is not None,
-#                     reason='Skip install on Release and Beta Firefox.')
-# @pytest.mark.nondestructive
-# def test_install_and_enable(base_url, selenium, firefox, notifications):
-#     Home(selenium, base_url).open()
-#     experiments = Experiments(selenium, base_url)
-#     experiment = experiments.find_experiment(experiment='Dev Example')
-#     experiment.install_and_enable()
-#     firefox.browser.wait_for_notification(
-#         notifications.AddOnInstallConfirmation).install()
-#     firefox.browser.wait_for_notification(
-#         notifications.AddOnInstallComplete).close()
-#     firefox.browser.wait_for_notification(
-#         notifications.AddOnInstallConfirmation).install()
-#     firefox.browser.wait_for_notification(
-#         notifications.AddOnInstallComplete).close()
-#     exp_detail = Detail(selenium, base_url)
-#     assert exp_detail.email_popup.is_popup_displayed()
-#     exp_detail.email_popup.close()
-#     assert exp_detail.enabled_popup.is_popup_displayed()
+@pytest.mark.skipif(os.environ.get('SKIP_INSTALL_TEST') is not None,
+                    reason='Skip install on Release and Beta Firefox.')
+@pytest.mark.nondestructive
+def test_install_and_enable(base_url, selenium, firefox, notifications):
```

Comment:
Looking the log it seems as if the Dev Example gets enabled but ```experiment.install_and_enable()``` is looking for something else. Specifically its within the ```self.wait.until()``` found in that class. I'd check there and make sure whatever it is waiting for matches the new page/styling.

Reply:
Maybe this got change? https://github.com/mozilla/testpilot/blob/master/frontend/test/ui/pages/desktop/detail.py#L45

Reply:
Seems to be the same, was just moved into ExperimentButtons https://github.com/mozilla/testpilot/blob/master/frontend/src/app/containers/ExperimentPage/ExperimentButtons.js#L143